packages/http/src/validator/validators/body.ts

```diff
@@ -19,19 +23,33 @@ export class HttpBodyValidator implements IHttpValidator<any, IMediaTypeContent>
       return [];
     }
 
-    const encodingStyle = get(content, 'encodings[0].style');
+    if (mediaType && typeIs.is(mediaType, 'application/x-www-form-urlencoded')) {
```

Comment:
I _think_ we can avoid the first check?
```suggestion
    if (typeIs.is(mediaType, 'application/x-www-form-urlencoded')) {
```